Configure.pl

```diff
@@ -549,6 +548,37 @@ sub uniq {
 build::probe::has_isinf_and_isnan(\%config, \%defaults);
 build::probe::unaligned_access(\%config, \%defaults);
 build::probe::ptr_size(\%config, \%defaults);
+build::probe::stdatomic(\%config, \%defaults);
+
+$config{use_mimalloc} = $args{mimalloc};
+if (!defined $config{mimalloc}) {
```

Comment:
Whoops, like one was missed.

Reply:
Bother yes, well spotted. Clearly the order in https://colabti.org/irclogger/irclogger_log/moarvm?date=2022-01-09#l62 was wrong. I guess next time it ought be

1. make coffee
2. re-read commit again
3. only then (if good) push